moveit_planners/ompl/ompl_interface/test/test_planning_context_manager.cpp

```diff
@@ -0,0 +1,343 @@
+    goal_state.setToDefaultValues();
+    goal_state.setJointGroupPositions(joint_model_group_, goal);
+    moveit_msgs::Constraints joint_goal =
+        kinematic_constraints::constructGoalConstraints(goal_state, joint_model_group_, 0.001);
+    request.goal_constraints.push_back(joint_goal);
+
+    return request;
+  }
+
+  /** \brief Helper function to create a position constraint. **/
+  moveit_msgs::PositionConstraint createPositionConstraint(std::array<double, 3> position,
+                                                           std::array<double, 3> dimensions)
+  {
+    shape_msgs::SolidPrimitive box;
+    box.type = shape_msgs::SolidPrimitive::BOX;
+    box.dimensions.resize(3);
+    box.dimensions[shape_msgs::SolidPrimitive::BOX_X] = dimensions[0];
+    box.dimensions[shape_msgs::SolidPrimitive::BOX_Y] = dimensions[1];
+    box.dimensions[shape_msgs::SolidPrimitive::BOX_Z] = dimensions[2];
+
+    geometry_msgs::Pose box_pose;
+    box_pose.position.x = position[0];
+    box_pose.position.y = position[1];
+    box_pose.position.z = position[2];
+    box_pose.orientation.w = 1.0;
+
+    moveit_msgs::PositionConstraint position_constraint;
+    position_constraint.header.frame_id = base_link_name_;
+    position_constraint.link_name = ee_link_name_;
+    position_constraint.constraint_region.primitives.push_back(box);
+    position_constraint.constraint_region.primitive_poses.push_back(box_pose);
+
+    return position_constraint;
+  }
+
+  ompl_interface::ModelBasedStateSpacePtr state_space_;
+  ompl_interface::ModelBasedPlanningContextSpecification planning_context_spec_;
+  ompl_interface::ModelBasedPlanningContextPtr planning_context_;
+  planning_scene::PlanningScenePtr planning_scene_;
+
+  constraint_samplers::ConstraintSamplerManagerPtr constraint_sampler_manager_;
+
+  /** Ideally we add an IK plugin to the joint_model_group_ to test the PoseModel state space, using the pluginlib to
+   * load the default KDL plugin? **/
+  // std::shared_ptr<kinematics::KinematicsBase> ik_plugin_;
+
+  ros::NodeHandle node_handle_;
+};
+
+/***************************************************************************
+ * Run all tests on the Panda robot
+ * ************************************************************************/
+// class PandaTestPlanningContext : public TestPlanningContext
+// {
+// protected:
+//   PandaTestPlanningContext() : TestPlanningContext("panda", "panda_arm")
+//   {
+//   }
+// };
+
+// TEST_F(PandaTestPlanningContext, testSimpleRequest)
+// {
+//   // use the panda "ready" state from the srdf config as start state
+//   // we know this state should be within limits and self-collision free
+//   testSimpleRequest({ 0, -0.785, 0, -2.356, 0, 1.571, 0.785 }, { 0, -0.785, 0, -2.356, 0, 1.571, 0.685 });
+// }
+
+// TEST_F(PandaTestPlanningContext, testPathConstraints)
+// {
+//   testPathConstraints({ 0, -0.785, 0, -2.356, 0, 1.571, 0.785 }, { 0, -0.785, 0, -2.356, 0, 1.571, 0.685 });
+// }
```

Comment:
Why are these commented out?

Reply:
No reason, just stupidity. Thanks for commenting!